__tests__/utils.test.ts

```diff
@@ -11,17 +11,29 @@ import {
 } from '../src/utils';
 import { HIDDEN_MARKER } from '../src/constants';
 
-describe('isBotPr()', () => {
-  it('should return true for dependabot PR', () => {
-    expect(isBotPr('dependabot')).toBeTruthy();
+jest.spyOn(console, 'log').mockImplementation();
+
+describe('shouldSkipBranchLint()', () => {
+  it('should recognize bot PRs', () => {
+    expect(shouldSkipBranchLint('dependabot')).toBeTruthy();
   });
 
-  it('should return false for non bot PR', () => {
-    expect(isBotPr('feature/awesomeNewFeature')).toBeFalsy();
+  it('should handle custom ignore patterns', () => {
+    expect(shouldSkipBranchLint('bar', '^bar')).toBeTruthy();
+    expect(shouldSkipBranchLint('foobar', '^bar')).toBeFalsy();
+
+    expect(shouldSkipBranchLint('bar', '[0-9]{2}')).toBeFalsy();
+    expect(shouldSkipBranchLint('bar', '')).toBeFalsy();
+    expect(shouldSkipBranchLint('foo', '[0-9]{2}')).toBeFalsy();
+    expect(shouldSkipBranchLint('f00', '[0-9]{2}')).toBeTruthy();
```

Comment:
can you add one more test for `release` and `production` branches in the pattern

Reply:
Added!

Reply:
Can you check `0295c34`

Reply:
:medal_sports: